README.md

```diff
@@ -27,29 +27,25 @@ The following steps will check out the repository, install the dependencies need
 ```bash
 $ git clone https://github.com/apoclyps/my-dev-space
 $ cd my-dev-space
-$ docker-compose -f docker-compose-dev.yml up -d
+$ make start
```

Comment:
It's worth creating a copy of the README for `Setting up a local development environment` with all the docker commands and dropping it into the `docs` directory for prosperity.